src/Microsoft.Azure.WebJobs.Shared.Storage/Extensions.cs

```diff
@@ -0,0 +1,75 @@
+﻿using Microsoft.WindowsAzure.Storage;
+using Microsoft.WindowsAzure.Storage.Blob;
+using Microsoft.WindowsAzure.Storage.Shared.Protocol;
+using System;
+using System.IO;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+internal static class StroageExtensions
```

Comment:
spelling

Recommend adding new/separate extension classes for each of these 3 types so this file doesn't become a dumping ground.